app/src/main/java/com/hover/stax/navigation/AbstractNavigationActivity.java

```diff
@@ -82,15 +82,16 @@ protected NavController getNavController() {
     }
 
     public void checkPermissionsAndNavigate(int toWhere) {
```

Comment:
This looks wrong, why is permissions check commented?

Reply:
Thanks, was an oversight to refractor back.